misk/src/main/kotlin/misk/web/MiskWebModule.kt

```diff
@@ -150,7 +151,12 @@ class MiskWebModule(private val config: WebConfig) : KAbstractModule() {
 
   @Provides @Singleton
   fun provideJettyThreadPool(): QueuedThreadPool {
-    return config.jetty_max_thread_pool_size?.let { QueuedThreadPool(it) } ?: QueuedThreadPool()
+    val threadPool = QueuedThreadPool(config.jetty_max_thread_pool_size,
+        8,
```

Comment:
These are the defaults today that jetty uses when we don't specify things

Reply:
can we use named params here

Reply:
Added vals